### ISSUE TITLE
focus outline is broken on adjacent grouped buttons

### ISSUE THREAD

**Issue by rgavrilov** [twbs/bootstrap#17561]
http://jsfiddle.net/v5yzsb9k/

Click first Go! button.

On Chrome (45.0.2454.85 m) and IE (11.0.23) on Windows 7 64-bit focus outline is missing right border.

Because second Go! button has z-index set to 2, therefore covering outline of the preceding button.

Here is a culprit rule which changed from 3.3.2:

    .input-group-btn:last-child > .btn,
    .input-group-btn:last-child > .btn-group {
      z-index: 2; // added after 3.3.2
      margin-left: -1px;
    }

**Comment by twbs-lmvtfy** [comment]
Hi @rgavrilov!

You appear to have posted a live example (http://fiddle.jshell.net/v5yzsb9k/show/light/), which is always a good first step. However, according to [Bootlint](https://github.com/twbs/bootlint), **your example has some Bootstrap usage errors**, which might potentially be causing your issue:
* line 41, column 7: [E003](https://github.com/twbs/bootlint/wiki/E003): Found one or more `.row`s that were not children of a grid column or descendants of a `.container` or `.container-fluid`
* line 49, column 19: [E036](https://github.com/twbs/bootlint/wiki/E036): Having multiple `.btn`s inside of a single `.input-group-btn` is not supported

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)
